.github/workflows/plugin_upstream_base.yml

```diff
@@ -0,0 +1,49 @@
+---
+name: "Nautobot Plugin Upstream Monitor - Base"
+
+on:
+  workflow_call:
+    inputs:
+      invoke_context_name:
+        required: true
+        type: string
+      plugin_name:
+        required: true
+        type: string
+
+env:
+  PYTHON_VERSION: "3.7"
+
+jobs:
+  unittest:
+    strategy:
+      fail-fast: false
+      matrix:
+        nautobot-version: ["next", "develop", "stable", "1.4", "1.3"]
```

Comment:
This is my only concern here which mirrors that of @glennmatthews in the other example repo (https://github.com/nautobot/nautobot-plugin-netutils/pull/2): I don't see that much value in constantly testing against `stable`, which won't change often, and old versions like `1.3` that won't be changing at all.

I think the ability to also be able to run this workflow from a plugin's repo manually against a given target version of Nautobot could be very appealing, however.

If we were to automate the runs, I think weekly or even bi-weekly would be more than sufficient. And I also think that running against a matrix of targeted versions on releases might also be a good idea?